commands.js

```diff
@@ -272,6 +272,26 @@ exports.commands = {
 	},
 	backhelp: ["/back - Unblocks challenges and/or private messages, if either are blocked."],
 
+	rank: function (target, room, user) {
+		if (!target) target = user.name;
```

Comment:
If you're not going to add `Ladders.visualizeAll` to `ladders-remote.js`, the next best way is to just return an error message here if `Ladders.visualizeAll` doesn't exist.